andela/hc-wits-kla
# #158175316 Notify the user when a job is running too often.

### What does this PR do?
- Ensure that the user is notified when a job is running too often
### How should this be manually tested?
- Clone the branch, `git clone https://github.com/andela/hc-wits-kla/tree/ft-user_notification-158175316`
- In your terminal, `cd hc-wits-kla`
- In the project directory, activate the environment, `source .env`
- Install requirements `pip install requirements.txt`
- Make migrations `python manage.py makemigrations`
- Run migrations `python manage.py migrate`
- Serve the application `python manage.py runserver`
- In the browser, visit, `127.0.0.1:8000/checks/`
- In your terminal, listen for alerts `python manage.py sendalerts`
- In your browser create a check, set **Period** to 2 minutes and **Grace Time** to 1 minute  and copy-paste it in another tab, and then again in another tab, hitting enter to load the page in each case. Do this in less than a minute.
- Go to the console and wait, the out put should be` -- PINGED QUITE OFTEN --`
### Relevant Pivotal Tracker story.
[#158175316](https://www.pivotaltracker.com/story/show/158175316)
